.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+### 🖼️ Screenshots
+
+🏚️ Before | 🏡 After
+---|---
+B | A
+
+
+### 🚧 TODO
+
+- [ ] ...
+
+### 🏁 Checklist
+
+- [ ] ⛑️ Tests (unit and/or integration) are included or not needed
+- [ ] 🔖 Capability is checked or not needed 
+- [ ] 🔙 Backport requests are created or not needed: `/backport to stable-xx.x`
+- [ ] 📅 Milestone is set
```

Comment:
IIRC people outside of the Org can not set milestones, so not sure this makes a lot of sense.

Reply:
i would say not all of the points from the checklist must be checked by community members.
so for backport requests and milestone i suggest to add "(should be done by nextcloud employees)" ?

Reply:
Best to add an not rendered comment about the optional items on the list, same is true for screenshots

Reply:
But then people think they can skip it although they should just do 2 screenshots.
Let's keep it and the milestone for now and see if there is community feedback afterwards

Reply:
Makes sense! Release early, release often :)